iexec-sms-library/src/main/java/com/iexec/sms/api/TeeSessionGenerationError.java

```diff
@@ -50,6 +50,8 @@ public enum TeeSessionGenerationError {
     SECURE_SESSION_STORAGE_CALL_FAILED,
     SECURE_SESSION_GENERATION_FAILED,
     SECURE_SESSION_NO_TEE_FRAMEWORK,
+    SECURE_SESSION_NO_TEE_ENCLAVE_CONFIGURATION,
```

Comment:
We will prefer removing this and use the existing `APP_COMPUTE_NO_ENCLAVE_CONFIG` instead